README.md

```diff
@@ -187,6 +187,20 @@ finetuner.encode(model=model, data=da)
 da.summary()
 ```
 
+When encoding, you can provide data either as a DocumentArray or a list. Since the modality of your input data can be inferred from the model being used, there is not need to provide any additional information besides the content you want to encode:
+
+```python
+import finetuner
+from docarray import Document, DocumentArray
+
+da = ['~/Pictures/your_img.png']
```

Comment:
let's avoid `da` as name since it's short for document array, something like `images` or `imgs` or `img_urls` would be better